.github/workflows/test.yml

```diff
@@ -0,0 +1,18 @@
+name: Primer Components
```

Comment:
Should we give these workflows different names? Maybe "Test" and "Publish."

Reply:
GitHub uses this string as the first part of the name, and the job name as the second part:

![image](https://user-images.githubusercontent.com/189606/82266445-9240a480-991e-11ea-85a1-d6529dabae4d.png)